### PR TITLE
fix: updated donation progress with heading

### LINKED ISSUE #54156: Donation progress bar UI/UX
### Describe the Issue

The progress bar animation gives off a stuck-loading vibe. I'd suggest not animating the progress and just filling it in.

As mentioned in the forum thread this can be labeled better as well. E.g. "Progress toward donation goal" or whatever.

### Affected Page

https://www.freecodecamp.org/learn

### Steps to Reproduce

2. Click on '...'
3. Scroll down to '...'

### Expected behavior

Less potentially confusing UX/UI

### Screenshots
progress-bar

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

Forum: https://forum.freecodecamp.org/t/freecodecamp-wont-load/680791

### PR DESCRIPTION
**Pull request by JeevaRamanathan**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #54156 

Updated Block Progress Bar with heading
image

### CODE REVIEW

File: client/src/components/Intro/learn-alert.tsx [hunk 1]
```diff
@@ -22,12 +22,15 @@ const LearnAlert = ({
     <Alert variant='info' className='annual-donation-alert'>
       {value && (
         <>
+          <div className='text-center'>
+            <h2>{t('learn.donation-heading')}</h2>
+            <h3>{`${value}%`}</h3>
```
Comment (bbsmooth) [inline client/src/components/Intro/learn-alert.tsx:27]:
```suggestion
            <p>{`${value}%`}</p>
```
This isn't acting as a heading.

**Comment by lasjorg** [comment]
I haven't checked, but are we sure `.progress-bar-percent` isn't used elsewhere?

I'm not saying the linear-gradient with the animation is working well if it is used elsewhere, but changing it for all instances might not be the correct thing to do here.

Maybe we should make a new class or a modifier class (so it can exist with and without the gradient).

**Comment by JeevaRamanathan** [comment]
Oops! Somehow I have missed that, animation has been removed now.
<hr/>

Ok, let's see if we hear any feedback from maintainers

**Comment by lasjorg** [comment]
BTW, how does it look with the paragraph element?

I didn't check, but I would assume the element needs some styles to be larger and have more weight?

**Comment by jeem5110** [comment]
Updated view with `p` element
image

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
image

The new text will need a specific colour set so it does not become illegible in night mode.

**Comment by jeem5110** [comment]
@naomi-lgbt, it has been updated with a specific color, and I hope the translation text, which is showing as "learn.donation-heading," will map with its corresponding value once it's live.

**Comment by JeevaRamanathan** [comment]
@naomi-lgbt, it has been updated with a specific color, and I hope the translation text, which is showing as "learn.donation-heading," will map with its corresponding value once it's live.

**Comment by naomi-lgbt** [comment]
> which is showing as "learn.donation-heading," will map with its corresponding value once it's live.

Yeah, that only shows because I was too lazy to clean the local files when I was switching branches.

**Review by naomi-lgbt — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
LGTM 🎉 

Hey @ahmaxed, do you mind taking a look at this redesign before we merge?

**Comment by ahmaxed** [comment]
This is the final state. Removed the hr as requested. Adding an animation would be nice. 

Screenshot 2024-04-27 at 9 03 44 PM

**Pull request merged**